### ISSUE TITLE
Metadata browse index sorting options not functional

### ISSUE THREAD

**Issue by Atmire-Kristof** [DSpace/DSpace#8897]
**Describe the bug**
The configuration for the browse index of a metadata field, e.g. `webui.browse.index.2 = author:metadata:dc.contributor.*\,dc.creator:text` should be able to accept a default sort option by configuring a sort direction and option directly after the index configuration, e.g. `:asc:dateissued`, but when done so, no default sorting is applied when requesting items from that browse endpoint.

As per the documentation above the browse index configuration in `dspace.cfg`:

```
#   The two last parts of the configuration are optional, and specifies the default ordering
#   for the index - whether it is ASCending (the default, and best for text indexes), or
#   DESCending (useful for dates - ie. most recent submissions) - and the sort option to use.
#   If you want to define the sort option you must define order as well.
```

Additional issue on the Angular side of this:
When visiting the browse page of the relevant index, e.g. `/browse/author`, the page loads indefinitely because it tries to instantly access the items endpoint, when it should only access the entries endpoint first (items endpoint requires a filterValue parameter, but we haven't even clicked an author yet, so it's missing, resulting in a 500 error response)

**To Reproduce**
1. Configure a metadata browse index with sorting options in `dspace.cfg`, e.g. `webui.browse.index.2 = author:metadata:dc.contributor.*\,dc.creator:text:asc:dateissued` (this should sort the items on the browse-by-author page by their date issued, ascending)
2. Visit the items endpoint of the browse index for a known author, e.g. `/server/api/discover/browses/author/items?filterValue=De%20Langhe,%20Kristof`
3. The resulting items should be sorted by date issued, but aren't. Instead, they're still sorted by title.
4. Visit `/browse/author` in the UI
5. The page loads indefinitely and a 500 error response can be found in the network tab for the items endpoint on the browse index

**Expected behavior**
1. The resulting items on the browse endpoint are expected to be sorted by the given default sort option (e.g. date issued, as per example above)
2. The UI browse page should load correctly

**Comment by tdonohue** [comment]
@Atmire-Kristof : this looks, to me, like it might be a duplicate of #8651

**Comment by tdonohue** [comment]
Closing as this appears to be a duplicate of #8651.  If anyone wants to help out on fixing this, there still the #8871 PR which is in consideration for 8.0 (it didn't get enough reviews for 7.6).... it might be plausible to backport the fixes to 7.6.x as well.
